cmd/api-datatypes.go

```diff
@@ -18,20 +18,27 @@ package cmd
 
 import (
 	"encoding/xml"
+	"time"
 )
 
 // DeletedObject objects deleted
 type DeletedObject struct {
-	DeleteMarker          bool   `xml:"DeleteMarker,omitempty"`
-	DeleteMarkerVersionID string `xml:"DeleteMarkerVersionId,omitempty"`
-	ObjectName            string `xml:"Key,omitempty"`
-	VersionID             string `xml:"VersionId,omitempty"`
+	DeleteMarker                  bool   `xml:"DeleteMarker,omitempty"`
```

Comment:
Can you please add comments to explain `DeleteMarkerVersionID` and other fields in here.